recommendation/system.web.json

```diff
@@ -105,25 +109,28 @@
               "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
             }
           ],
-          "Description": "Replace HttpContextBase with HttpContext. Replace System.Web namespace with Microsoft.AspNetCore.Http",
+          "Description": "Porting assistant will replace HttpContextBase with HttpContext. In addition, System.Web namespace will be replaced with Microsoft.AspNetCore.Http",
```

Comment:
"Description": "Replace HttpContextBase with HttpContext. Replace System.Web namespace with Microsoft.AspNetCore.Http",

"Description": "Porting assistant will replace HttpContextBase with HttpContext. In addition, System.Web namespace will be replaced with Microsoft.AspNetCore.Http",

Suggested:
Replace HttpContextBase from namespace 'System.Web' with HttpContext from namespace 'Microsoft.AspNetCore.Http'